src/d3a/models/strategy/external_strategies/load.py

```diff
@@ -398,35 +396,54 @@ def event_tick(self):
         for req in self.pending_requests:
             if req.request_type == "set_energy_forecast":
                 self._set_energy_forecast_impl(req.arguments, req.response_channel)
+            elif req.request_type == "set_energy_measurement":
+                self._set_energy_measurement_impl(req.arguments, req.response_channel)
 
         self.pending_requests = deque(
             req for req in self.pending_requests
-            if req.request_type not in "set_energy_forecast")
```

Comment:
How was this working before? O.o

Reply:
Haha, indeed. I dont understand either. Maybe this was the reason why [this](https://github.com/gridsingularity/d3a/blob/30538c352420bc0b1e5b8b6ae2a2c3c2125a6620/src/d3a/models/strategy/external_strategies/pv.py#L389)  hat to be added.